docs/source/best-practices.rst

```diff
@@ -100,7 +109,7 @@ start to overwhelm the scheduler.
 
 There are a few things you can do to address this:
 
--   Build smaller graphs.  You can do this by ...
+-   Build smaller graphs.  You can do this by:
```

Comment:
The indentation here looks a bit strange to me. What are your thoughts? 

The changes here don't impact the indentation that already exists on `main`, so I don't intend for this comment to be blocking

Reply:
Oh good catch! I can push up a small PR.